### PR TITLE
Rerender

### PR DESCRIPTION
**Pull request by ocefpaf**
Needed in https://github.com/conda-forge/qgis-feedstock/pull/101

### CODE REVIEW

**Comment by gillins** [comment]
@ocefpaf what was the solution for the `libGL.so` error? Do we need a `yum_requirements.txt`?

```
x86_64-conda_cos6-linux-gnu-c++: error: /home/conda/feedstock_root/build_artifacts/qtlocation_1573588541126/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/bin/../x86_64-conda_cos6-linux-gnu/sysroot/usr/lib64/libGL.so: No such file or directory
```

**Comment by gillins** [comment]
@conda-forge-admin, please rerender

File: .scripts/build_steps.sh [hunk 1]
```diff
@@ -26,6 +26,14 @@ setup_conda_rc "${FEEDSTOCK_ROOT}" "${RECIPE_ROOT}" "${CONFIG_FILE}"
 
 source run_conda_forge_build_setup
 
+
+# Install the yum requirements defined canonically in the
+# "recipe/yum_requirements.txt" file. After updating that file,
+# run "conda smithy rerender" and this line will be updated
+# automatically.
+/usr/bin/sudo -n yum install -y mesa-libGL
```
Comment (ocefpaf) [inline .scripts/build_steps.sh:34]: Looks like this did not work :-/

**Comment by ocefpaf** [comment]
Failing with:

```
rm -f libQt5PositioningQuick.so.5.12.5 libQt5PositioningQuick.so libQt5PositioningQuick.so.5 libQt5PositioningQuick.so.5.12
$BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-c++ -Wl,--no-undefined -Wl,--version-script,QtPositioningQuick.version -Wl,-O1 -fuse-ld=gold -Wl,--disable-new-dtags -Wl,-z,origin -Wl,-rpath,\$ORIGIN -shared -Wl,-Bsymbolic-functions -Wl,-soname,libQt5PositioningQuick.so.5 -o libQt5PositioningQuick.so.5.12.5 .obj/qdeclarativeposition.o .obj/qdeclarativepositionsource.o .obj/qquickgeocoordinateanimation.o .obj/moc_qdeclarativeposition_p.o .obj/moc_qdeclarativepositionsource_p.o .obj/moc_qquickgeocoordinateanimation_p.o  -L$PREFIX/lib -L$PREFIX/x86_64-conda_cos6-linux-gnu/sysroot/usr/lib64 $SRC_DIR/build/lib/libQt5Positioning.so $PREFIX/lib/libQt5Quick.so $PREFIX/lib/libQt5Gui.so $PREFIX/lib/libQt5Qml.so $PREFIX/lib/libQt5Network.so $PREFIX/lib/libQt5Core.so -lpthread $PREFIX/bin/../x86_64-conda_cos6-linux-gnu/sysroot/usr/lib64/libGL.so   
x86_64-conda_cos6-linux-gnu-c++: error: /home/conda/feedstock_root/build_artifacts/qtlocation_1573598660246/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/bin/../x86_64-conda_cos6-linux-gnu/sysroot/usr/lib64/libGL.so: No such file or directory
make[2]: *** [../../lib/libQt5PositioningQuick.so.5.12.5] Error 1
make[2]: Leaving directory `$SRC_DIR/build/src/positioningquick'
make[1]: *** [sub-positioningquick-make_first] Error 2
make[1]: Leaving directory `$SRC_DIR/build/src'
make: *** [sub-src-make_first] Error 2
```

TL;DR it cannot find `...placehold_/bin/../x86_64-conda_cos6-linux-gnu/sysroot/usr/lib64/libGL.so`. @isuruf are we missing something obvious here?

**Comment by isuruf** [comment]
I thought this was fixed in https://github.com/conda-forge/qt-feedstock/pull/124. Is it using the latest qt build?

**Comment by gillins** [comment]
Looks to be the latest build to me...

**Comment by ocefpaf** [comment]
> I thought this was fixed in [conda-forge/qt-feedstock#124](https://github.com/conda-forge/qt-feedstock/pull/124). Is it using the latest qt build?

Maybe we need to revisit that PR.

**Comment by gillins** [comment]
Are we still not sure what's going on here?

**Comment by isuruf** [comment]
Nope, but as a workaround you can add,
```
build:
  merge_build_host: True   # [linux]
```

**Comment by ocefpaf** [comment]
Windows should pass if we restart in ~40 minutes due to the lack of jpeg 9c on Windows.

**Comment by ocefpaf** [comment]
Thanks @gillins and @isuruf!
